research/notes.org

```diff
@@ -97,9 +117,10 @@ assertions on the type of that variable.
 :PROPERTIES:
 :CUSTOM_ID: sec:nabl
 :END:
-With /NaBL/ (pronounced /enable/), name binding and scoping rules can
-be specified declaratively\nbsp\cite{KonatKWV12}. Here is an example
-of the name binding and scoping rules for a class, from the /paplj/
+With /NaBL/ (pronounced /enable/), name binding and scoping can be
+specified declaratively using AST-based
```

Comment:
`AST-based` -> `syntax-directed`? You just introduced this term so might as well use it.